ts/WoltLabSuite/Core/Component/Interaction/LegacyDboAction.ts

```diff
@@ -31,21 +34,25 @@ async function handleDboAction(
     .payload(confirmationResult.reason ? { reason: confirmationResult.reason } : {})
     .dispatch();
 
-  if (confirmationType == ConfirmationType.Delete) {
+  if (interactionEffect === InteractionEffect.ReloadItem) {
```

Comment:
I would like to see this being a `switch` instead in order to handle unknown effects by raising an exception.